xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/Content.xml

```diff
@@ -39,6 +39,7 @@
   <content>{{velocity}}
 #if ($type == 'edit')
   #set ($className = $object.getxWikiClass().name)
+  #set ($id = $escapetool.xml("${prefix}${name}"))
```

Comment:
> That escaping looks wrong, as the $id variable is later passed to $services.edit.syntaxContent.wysiwyg, and I would be surprised if it expected an escaped value.

You're right, thanks for finding this :+1: From what I remember, it would be incorrect for such a service to expect the value escaped, so we can assume we shouldn't escape it. If there's an issue, it should be solved on the service provider instead.

Addressed in https://github.com/xwiki/xwiki-platform/pull/3584/commits/f0a99b80e7c9c134a3228ca4e2f8ec3145195318 :+1: